src/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios"
+
+const newsApi = axios.create({
+    baseURL: 'https://nc-news-ms3a.onrender.com/api'
+})
+
+export const getArticles = () => {
+    return newsApi.get('/articles')
+}
```

Comment:
Great use of an axios instance here, very clean! Will really come into play when you've got a lot more endpoints